src/Python-2.0.1.xml

```diff
@@ -30,37 +30,37 @@
                2002, 2003, 2004, 2005, 2006</alt>
                Python Software Foundation; All Rights Reserved" are retained in
-               Python <alt match="2.0.1|3.10.6| " name="version">2.0.1</alt> alone or in any derivative version prepared by Licensee.
+               Python <alt match="(([0-9]+)\.([0-9]+)\.([0-9]+))?" name="version">2.0.1</alt> alone or in any derivative version prepared by Licensee.
             </item>
             <item>
                <bullet>3.</bullet>
                In the event Licensee prepares a derivative work that is
-               based on or incorporates Python <alt match="2.0.1|3.10.6| " name="version">2.0.1</alt> or any part thereof, and
+               based on or incorporates Python <alt match="(([0-9]+)\.([0-9]+)\.([0-9]+))?" name="version">2.0.1</alt> or any part thereof, and
                wants to make the derivative work available to others as
                provided herein, then Licensee hereby agrees to include in
-               any such work a brief summary of the changes made to Python <alt match="2.0.1|3.10.6| " name="version">2.0.1</alt>.
+               any such work a brief summary of the changes made to Python <alt match="(([0-9]+)\.([0-9]+)\.([0-9]+))?" name="version">2.0.1</alt>.
             </item>
             <item>
                <bullet>4.</bullet>
-               PSF is making Python <alt match="2.0.1|3.10.6| " name="version">2.0.1</alt> available to Licensee on an "AS IS"
+               PSF is making Python <alt match="(([0-9]+)\.([0-9]+)\.([0-9]+))?" name="version">2.0.1</alt> available to Licensee on an "AS IS"
                basis. PSF MAKES NO REPRESENTATIONS OR WARRANTIES, EXPRESS
                OR IMPLIED. BY WAY OF EXAMPLE, BUT NOT LIMITATION, PSF
                MAKES NO AND DISCLAIMS ANY REPRESENTATION OR WARRANTY OF
                MERCHANTABILITY OR FITNESS FOR ANY PARTICULAR PURPOSE OR THAT
-               THE USE OF PYTHON <alt match="2.0.1|3.10.6| " name="version">2.0.1</alt> WILL NOT INFRINGE ANY THIRD PARTY RIGHTS.
+               THE USE OF PYTHON <alt match="(([0-9]+)\.([0-9]+)\.([0-9]+))?" name="version">2.0.1</alt> WILL NOT INFRINGE ANY THIRD PARTY RIGHTS.
             </item>
             <item>
                <bullet>5.</bullet>
                PSF SHALL NOT BE LIABLE TO LICENSEE OR ANY OTHER
-               USERS OF PYTHON <alt match="2.0.1|3.10.6| " name="version">2.0.1</alt> FOR ANY INCIDENTAL, SPECIAL, OR
+               USERS OF PYTHON <alt match="(([0-9]+)\.([0-9]+)\.([0-9]+))?" name="version">2.0.1</alt> FOR ANY INCIDENTAL, SPECIAL, OR
                CONSEQUENTIAL DAMAGES OR LOSS AS A RESULT OF MODIFYING,
-               DISTRIBUTING, OR OTHERWISE USING PYTHON <alt match="2.0.1|3.10.6| " name="version">2.0.1</alt>, OR ANY DERIVATIVE
+               DISTRIBUTING, OR OTHERWISE USING PYTHON <alt match="(([0-9]+)\.([0-9]+)\.([0-9]+))?"" name="version">2.0.1</alt>, OR ANY DERIVATIVE
```

Comment:
@jlovejoy looks like it's two quotes by mistake at the end of the `match` attribute, this might be causing the error?

Reply:
fixed